mfr/extensions/settings.py

```diff
@@ -0,0 +1,11 @@
+from mfr import settings
+
+config = settings.child('EXTENSION_CONFIG')
+
+# LOCAL_DEVELOPMENT must be set to true for extensions that produce a waterbutler URL in the
+# TEMPLATE to render, when using local docker development environment. Call will be coming from
+# browser instead of docker container.
+
+LOCAL_DEVELOPMENT = config.get_bool('LOCAL_DEVELOPMENT', 0)
```

Comment:
Nitpick: `0` ==> `False`

Reply:
Everything else looks good, so I'll take care of it.  But generally the settings should follow the idiom they're defined in. This is a python config file, so `True` and `False` are appropriate.  The `.docker-compose.*.env` files are env files, so `1` and `0` are appropriate for them.